package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@testing-library/react-native",
-  "version": "9.0.0-alpha.0",
+  "version": "9.0.0",
```

Versions are commited separately, let's not tie it to specific changes.
```suggestion
  "version": "9.0.0-alpha.0",
```